### PR TITLE
fix: broad expand with more than 50 columns

### PR DESCRIPTION
**Pull request by johannes-vogel**
We've detected in the runtime tests that broad expands were not yet working properly with udf `json_merge`. It must be parsed with one of the sqlite json functions to be proper input for json_group_array.
-> Makes the workaround even uglier...

### CODE REVIEW

File: test/compliance/resources/db/complex/index.cds [hunk 1]
```diff
@@ -1 +1,13 @@
...
+  key ID : Integer;
+  title  : String(111);
+  author : Association to Authors;
+}
+
+entity Authors {
+  key ID : Integer;
+  name   : String(111);
+  books  : Association to many Books on books.author = $self;
+}
```
Comment (patricebender) [inline test/compliance/resources/db/complex/index.cds:13]:
```suggestion
}

```

File: test/compliance/resources/db/data/complex.Authors.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+ID;name
+1;Emily
```
Comment (patricebender) [inline test/compliance/resources/db/data/complex.Authors.csv:2]:
```suggestion
1;Emily

```

File: test/compliance/resources/db/data/complex.Books.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+ID;title;author_ID
+1;Wuthering Heights;1
```
Comment (patricebender) [inline test/compliance/resources/db/data/complex.Books.csv:2]:
```suggestion
1;Wuthering Heights;1

```

File: test/compliance/SELECT.test.js [hunk 4]
```diff
@@ -159,6 +159,34 @@ describe('SELECT', () => {
...
+      const cqn = {
+        SELECT: {
+          from: { ref: ['complex.Authors'] },
+          columns: [{ ref: ['ID']}, { ref: ['name']}, { ref: ['books'], expand: ['*', ...nulls(197)]}]
+        },
+      }
+
+      const res = await cds.run(cqn)
+      // ensure that all null values are returned and as json format
+      assert.strictEqual(Object.keys(res[0].books[0]).length, 200)
```
Comment (patricebender) [inline test/compliance/SELECT.test.js:173]: from the comment it should be checked whether the `nulls` is returned in json format. Is the check `.length` enough to make sure that this holds true?

Comment (johannes-vogel) [reply test/compliance/SELECT.test.js:173]: I'll adapt the comment. only the length is important.

**Review by patricebender — APPROVED** [review]

**Pull request merged**